opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -130,7 +130,7 @@ def __init__(
         resource: None = None,  # TODO
         attributes: types.Attributes = None,  # TODO
         events: Sequence[trace_api.Event] = None,  # TODO
-        links: Sequence[trace_api.Link] = None,  # TODO
+        links: Sequence[trace_api.Link] = None,
```

Comment:
Since `start_span` uses `()` as a default instead of `None`, maybe use `()` here too? Also we have a `Sequence` not an `Optional[Sequence]` anyway.